packages/cozy-stack-client/src/AppCollection.js

```diff
@@ -11,7 +11,12 @@ import { FetchError } from './errors'
 export const APPS_DOCTYPE = 'io.cozy.apps'
 
 export const normalizeApp = (app, doctype) => {
-  return { ...app, ...normalizeDoc(app, doctype), ...app.attributes }
+  return {
+    ...app,
+    ...normalizeDoc(app, doctype),
+    ...app.attributes,
+    id: app.id // ignores any 'id' attribute in the manifest
+  }
```

Comment:
Let's spread the app.attributes before app and normalizeDoc(app, doctype). Like that, if an entry is available in both, then the one at the first level (aka app) will be used. Like that no need to write id: app.id anymore and so on

Reply:
ping @doubleface

Reply:
done. Sorry for the very long delay